Makefile

```diff
@@ -35,7 +35,7 @@ container: build
 	docker build -t ${REGISTRY}/kube-state-metrics:$(TAG) .
 
 push: container
-	gcloud docker -- push ${REGISTRY}/kube-state-metrics:$(TAG)
```

Comment:
As I don't use this make target, we can leave this for the google people as a help, as the image should still be published on gcr.

Reply:
done.